### PR TITLE
Replaced get_shape() to shape.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
This is the recommended method.

### CODE REVIEW

File: tensorflow/python/keras/optimizer_v2/adadelta_test.py [hunk 1]
```diff
@@ -75,16 +75,16 @@ def doTestBasic(self, use_resource=False, use_callable_params=False):
             slot = [None] * 2
             slot_update = [None] * 2
             slot[0] = adadelta_opt.get_slot(var0, "accum_grad")
-            self.assertEqual(slot[0].get_shape(), var0.get_shape())
+            self.assertEqual(slot[0].shape, var0.get_shape())
```
Comment (omalleyt12) [inline tensorflow/python/keras/optimizer_v2/adadelta_test.py]: Why not here as well?

**Review by omalleyt12 — COMMENTED** [review]
Thanks for the PR!

File: tensorflow/python/keras/optimizer_v2/adadelta_test.py (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/python/keras/optimizer_v2/adadelta_test.py]: @omalleyt12 , thanks for the review, i have updated the code as per your suggestion. Kindly check and approve.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@omalleyt12 , thanks for the review, i have updated the code as per your suggestion. Kindly check and approve.

Regards
Amit

**Review by omalleyt12 — APPROVED** [review]

**Pull request merged**